### DISCUSSION TITLE
Welcome to Basic Car Maintenance Discussions!

### DISCUSSION THREAD

**Discussion by mikaelacaron** [mikaelacaron/Basic-Car-Maintenance#48] [Announcements]
## 👋 Welcome!

I'll be sharing updates here about the repo throughout Hacktoberfest (for example, if I decide to delete all the data in the database) or something similar that all contributors should be aware of.

I'll also be using Discussions as a place to connect with other contributors to this repo. I hope that you:
* Ask questions you’re wondering about (if it's more general, if it's connected to an issue, it's better to comment there)
* Share ideas (make an issue first if it's something small, but if it's more abstract make a new discussion here)
* Engage with other community members
* Welcome others and are open-minded. Remember that this is a welcoming community to developers of all experience levels!

To get started, comment below with an introduction of yourself and what iOS 17 feature you'd be interested in seeing in the codebase!

**Comment by harrythewise** [comment]
I guess I will take the priviledge of a first introduction. So Hi, my name is Jan and I’m currently a last year student doing Master’s in Software Engineering. This is my first year of Hacktober and I’m very glad I took part in it, because I realized how much I enjoying contributing to help others! 

If you want to know more about me, check my personal pages or hit me up on socials, which are available on my GitHub page :)

Looking forward to meeting y’all!

**Reply by mikaelacaron** [reply]
Thanks for kicking off the comments! :heart:

**Comment by horacio3m** [comment]
Firstly, I would like to introduce myself: My name is Horacio, I am 30 years old, and I am studying iOS development with SwiftUI. I would like to know if there are any issues or something I can contribute to in order to join this open source community with you, something at a beginner level..

**Reply by mikaelacaron** [reply]
Hi Horacio! Thanks for wanting to contribute! I've been thinking on more issues to add because they're all taken. I'll add some new ones today / tomorrow. (and post in the discussion announcements)

I'm labeling some `first-timers-only` that are specifically for anyone new who hasn't contributed to open source before.

**Reply by horacio3m** [reply]
ok! Im really excited to contribute. Ill be waiting

**Comment by thx1766** [comment]
Hi! My name is Nate, and I'm interested in contributing to this project!

I'm 34 and professionally work as a back end C# developer, though I've been dabbling in swift the last two years.

If there aren't open issues to be assigned and its in scope, I'm interested in adding the ability to track refueling (price/cost/amount) to chart efficiency etc.
